fighthealthinsurance/management/commands/launch_prefetch_actor.py

```
@@ -0,0 +1,49 @@
+            print(f"Pre-fetch task started: {task}")
+
+            # Wait for task to complete (with timeout)
+            # This is non-blocking for deployment - if it times out, we continue
+            timeout = 600  # 10 minutes
+            print(f"Waiting up to {timeout}s for pre-fetch to complete...")
+
+            ready, _ = ray.wait([task], timeout=timeout)
+
+            if ready:
+                result = ray.get(ready[0])
+                print(f"Pre-fetch completed successfully!")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)